src/hotspot/share/runtime/thread.inline.hpp

```diff
@@ -26,7 +26,7 @@
 #ifndef SHARE_RUNTIME_THREAD_INLINE_HPP
 #define SHARE_RUNTIME_THREAD_INLINE_HPP
 
-#include "runtime/javaThread.hpp"
+#include "thread.hpp"
```

Comment:
We tend to include the directory even if it is implied by the current file's location.

```suggestion
#include "runtime/thread.hpp"
```